src/zenml/integrations/aws/__init__.py

```diff
@@ -33,13 +33,9 @@ class AWSIntegration(Integration):
     """Definition of AWS integration for ZenML."""
 
     NAME = AWS
-    # We need to pin protobuf to a version <=4 here, as this sagemaker release
-    # does not pin it. They fixed this in a later version, so we can probably
-    # remove this once we update the sagemaker version.
     REQUIREMENTS = [
         "boto3==1.21.0",
-        "sagemaker==2.82.2",
-        "protobuf>=3.1,<4.0.0",
+        "sagemaker==2.117.0",
```

Comment:
those look ok to me @strickvl , but we won't know for sure until they are all installed together with the other integrations. My reason for bumping up these dependencies was to avoid conflicts with other integrations.